### ISSUE TITLE
Node.JS 

### ISSUE THREAD

**Issue by DocBitsSchubser** [cztomczak/phpdesktop#296]
Is it possible to install Node.JS under PhpDesktop? If so what is the best way to realize this.

**Comment by coderkoala** [comment]
Unfortunately not. It's configured under apache, you'll need something else. Try Electron.

**Comment by terremoth** [comment]
This questions seems a lot weird. 
Why would you want to install nodejs with PHP? 
Doesn't make sense, since both technologies are server side processors.  You either choose php or nodejs to process things.

If by, some reason, you need to install nodejs, you can make a php script that downloads nodejs and asks user to execute the installer (permission to install). But this would be a lot weird too;

**Comment by johnhart96** [comment]
I have to agree. Also if you want to use Node.JS then maybe you really want to be using Electron instead.

**Comment by cztomczak** [comment]
Looks like this was answered. If @mkoakintomiwa  you would like to put this answer to Knowledge Base wiki page feel free.
